org.eclipse.corrosion/grammar/rust.tmLanguage.json

```diff
@@ -0,0 +1,655 @@
+{
+	"information_for_contributors": [
```

Comment:
Should we keep this comment as it or should we mention that we pick the grammar from VSCode with a link to it?

Reply:
I would at least leave in the info to create issues on the original-original repository, but I wouldn't mind mentioning the direct VSCode source. @LucasBullen what is your opinion?

BTW: I don't know how MIT licensed content work, can we simply re-license this under EPL 2.0, because the licenses are compatible, or should I add license info in the file or to the feature project?

Reply:
> I would at least leave in the info to create issues on the original-original repository, but I wouldn't mind mentioning the direct VSCode source. 

Since we'll typically update by fetching the latest version from VSCode rather than by actually maintaining the translation ourselves, I think we need to mention VSCode there.
We can just put 2 links "File is taken from VSCode https://github.com/Microsoft/vscode/blob/8fdf170a0850c1cc027382f31650aaf300d3ae2a/extensions/rust/syntaxes/rust.tmLanguage.json as a translation of https://github.com/zargony/atom-language-rust/blob/master/grammars/rust.cson . Please contribute to those 2 files and then request Corrosion project to fetch latest version."

> BTW: I don't know how MIT licensed content work, can we simply re-license this under EPL 2.0, because the licenses are compatible, or should I add license info in the file or to the feature project?

I'll take care of asking the Eclipse IP team when we agree on a `information_for_contributors`.

Reply:
Looks good to me, I updated the PR accordingly.